src/locales/ko/achv.ts

```diff
@@ -264,6 +264,10 @@ export const PGMachv: AchievementTranslationEntries = {
   "MONO_FAIRY": {
     name: "설마 자시안으로?",
   },
+  "EEVEE_TRAINER": {
+    name: "이브이 트레이너",
+    description: "이브이 챌린지 모드 클리어.",
```

Comment:
```suggestion
    description: "이브이 모드 챌린지 클리어.",
```
@sodaMelon 
챌린지의 이름이 '이브이 모드'라 어순을 변경했습니다.